retry/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: retry
-version: 3.0.0+1
+version: 3.0.1+1
```

Comment:
Remove the `+1`.

Context: version `3.0.0` is different from `3.0.0+1` ... the `+1` part is call _build metadata_, see: https://semver.org/#spec-item-10

It's optional, and I only used it because the fix I made was documentation-only. hence, it didn't change the code... just how it was packaged.

In practice I should probably just have made a patch release instead, since that's less confusing.

Reply:
I got your point! If I know about it, it make sense to add "+1" for documentation. thx.
I fix it.